sympy/physics/control/tests/test_lti.py

```diff
@@ -1679,3 +1679,28 @@ def test_StateSpace_functions():
     assert ss3.input_matrix == Matrix([[0, 0], [1, 0], [0, 1], [0, 0]])
     assert ss3.output_matrix == Matrix([[0, 1, 0, 0], [0, 0, 1, 0]])
     assert ss3.feedforward_matrix == Matrix([[0, 0], [0, 1]])
+
+
+def test_Feedback_as_TransferFunction(): #Solves https://github.com/sympy/sympy/issues/26161
+    fd1 = Feedback(TransferFunction(s + 1, 1 , s), TransferFunction(s + 2, 1, s))
+    tf1 = TransferFunction(2 * (s+1), s**2 + 2*s + 1, s)
+    tf2 = TransferFunction(1, s + 1, s)
+    # Type Checking
+    assert isinstance(fd1, TransferFunction)
+    assert isinstance(fd1, Feedback)
+    assert not isinstance(tf1, Feedback)
+    # Operation checking
+    num, den = fd1.num, fd1.den
+    assert num == TransferFunction(s + 1, 1, s)
+    assert den == TransferFunction(s + 2, 1, s)
+    # Combining Operations of TransferFunction and Feedback
+    fd2 = Feedback(tf1*fd1, tf2)
+    assert fd2.doit().simplify() == TransferFunction(2*(s + 1)**2, 2*s + ((s + 1)*(s + 2) + 1)*(s**2 + 2*s + 1) + 2, s)
+    assert (tf1+fd1) == Parallel(TransferFunction(2*s + 2, s**2 + 2*s + 1, s),
```

Comment:
Hi, just for proof of correctness at my end can you point me to any links where these tests have been picked up from. I usually added MATLAB examples as my unit tests.

Reply:
Ohk, Sir I'll include the tests from MATLAB or other genuine sources and Update the tests by today.

Reply:
It's okay if you don't find relevant MATLAB examples, I would still try to use other online or textbook examples rather than making up tests on my own.

Reply:
I have gone over the tests, they look good to me!